src/control/Control.Layers.js

```diff
@@ -198,6 +198,8 @@ export var Layers = Control.extend({
 		var link = this._layersLink = DomUtil.create('a', className + '-toggle', container);
 		link.href = '#';
 		link.title = 'Layers';
+		link.setAttribute('aria-label', 'Layers');

```

Comment:
This should ideally not have both `aria-label` and `title`. I haven't raised a Leaflet issue on this yet... but see motivation in https://github.com/mapbox/mapbox-gl-js/issues/11036.

Reply:
So you suggest to remove `aria-label` from the PR? But will it then resolve the WCAG issues?